book/src/03_3_2_http_client.md

```diff
@@ -12,78 +10,63 @@ In this exercise, we'll write a small client that retrieves data over a HTTP con
 $ cargo doc --open
 ```
 
-✅ Add your network credentials to the `cfg.toml`.
+✅ Add your [network credentials](02_4_hello_board.md) to the `cfg.toml` as previously.
 
 ✅ Open the prepared project skeleton in `intro/http-client/exercise`. We'll read through the code together.
 
 ## Making a connection
 
 By default only unencrypted HTTP is available, which rather limits our options of hosts to connect to. We're going to use `http://neverssl.com/`.
 
-In `esp-idf`, HTTP client connections are managed by `http::client::EspHttpClient` in the `esp-idf-svc` crate. It implements the `http::client::Client` trait from `embedded-svc`, which defines functions for [HTTP request methods](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods) like `GET` or `POST`.
+In `esp-idf`, HTTP client connections are managed by `http::client::EspHttpClient` in the `esp-idf-svc` crate. It implements the `http::client::Client` trait from `embedded-svc`, which defines functions for [HTTP request methods](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods) like `GET` or `POST`. This is a good time to have a look at the documentation you opened with `cargo doc --open`.
+
+✅ Add the url `http://neverssl.com/` is in the main function. This is the address we will query.
 
-✅ Create a new `EspHttpClient` with default values.
+✅ Create a new `EspHttpClient` with default values (Try to find the method `new_default()` in the documentation).
```

Comment:
explanation: It's not necessary to check the doc if you already give the method away as there is no further configuration necessary. 

```suggestion
✅ Create a new `EspHttpClient` with default values. Look for a suitable instantiating method in the documentation.
```

Reply:
"instantiating method" -> "constructor"